Casks/l/lowtechguys-cling.rb

```diff
@@ -0,0 +1,27 @@
+cask "lowtechguys-cling" do
+  version "1.1"
+  sha256 "9f9b31d95fd4610735fa1afcc79d77b0162dfc8f5502bd24a4f08821d0062c71"
+
+  url "https://files.lowtechguys.com/releases/Cling-#{version}.dmg"
+  name "Low Tech Guys Cling"
```

Comment:
```suggestion
  name "Cling"
```

Reply:
The docs do allow for the inclusion of the vendor name where it helps with Disambiguation.
https://docs.brew.sh/Cask-Cookbook#stanza-name

Reply:
Based on their portfolio, the studio is called `The low-tech guys`. We could use the following to include the proper vendor name instead:

```suggestion
  name "The low-tech guys Cling"
```

Otherwise, happy to resolve.